renku/models/_jsonld.py

```diff
@@ -331,7 +331,10 @@ def from_jsonld(
             raise ValueError(data)
 
         if '@type' in data:
-            type_ = tuple(sorted(data['@type']))
+            type_ = data['@type']
```

Comment:
I find this `_jsonld` wrapper quite difficult to read. Could we add a commend what this change does? Just to make it a little bit easier for future to read/maintain this code.

Reply:
done